.github/workflows/builds.yml

```diff
@@ -7,26 +7,35 @@ env:
   GFORTRAN_PATH: "/usr/local/lib/gcc/11"
 
 jobs:
-  library-Build:
+  build:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-11]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
 
-      - name: Checkout tools repo
-        uses: actions/checkout@v2
-        with:
-          repository: HydrologicEngineeringCenter/dss-test-data
-          path: dss-test-data
-          
       - name: Build javaHeclib
         run: |
           sudo ln -s /usr/local/bin/gfortran-11 /usr/local/bin/gfortran
           cd ${{ github.workspace }}/heclib
           make
-          
+
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-11]
+    runs-on: ${{ matrix.os }}
+    if: startsWith(matrix.os, 'ubuntu')
```

Comment:
don't we want the tests to run on the macos runner?

Reply:
> don't we want the tests to run on the macos runner?

Of course.  The existing macOS workflow does not test the macOS runner.  Thinking to add it but testing is a pain.  The `startsWith` or `contains` does not like any iteration of ${{ matrix.os }}, '${{ matrix.os }}', or just matrix.os.  VS Code intelliSense doesn't have an issue but GitHub is angry with them.

Reply:
okay, no I had something similarish,  https://github.com/USACE/cwms-radar-api/blob/fc8a55a8970ebfad4a414e9dd4e484a7882a26c0/.github/workflows/build.yml#L111 but that doesn't really give you an example of the matrix there.